config/config.go

```diff
@@ -13,7 +13,7 @@ import (
 type Configuration struct {
 	Address  string      `json:"addr"`
 	Retain   int64       `json:"retain"`
-	Timeout  string      `json:"freq"`
+	Frequency string      `json:"frequency"`
```

Comment:
Looks like the formatter needs to be run here

Reply:
Should be good